setup.py

```diff
@@ -20,7 +20,8 @@
     ],
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'edx-django-oauth2-provider>=0.3.0,<1.0.0',
+        # TODO update with new django-oauth2-provider release
+        # 'edx-django-oauth2-provider>=0.3.0,<1.0.0',
```

Comment:
Likewise, I imagine this will be updated once edx/django-oauth2-provider#22 lands. I've commented there about updating the version number for that package so you can ensure it's installed here.